esp8266-arduino/ppd42ns-wificonfig-ppd-sds-dht/ppd42ns-wificonfig-ppd-sds-dht.ino

```diff
@@ -744,9 +757,24 @@ String wlan_ssid_to_table_row(const String& ssid, const String& encryption, cons
 }
 
 /*****************************************************************
+/* Webserver request auth: prompt for BasicAuth                              
+ *  
+ * -Provide BasicAuth for all page contexts except /values and images
+/*****************************************************************/
+void webserver_request_auth() {
+  debug_out(F("validate request auth..."),DEBUG_MIN_INFO,1);
+  if(www_basicauth_enabled) {
+    if(!server.authenticate(www_username, www_password))
```

Comment:
- `if (` instead of `if(`
- consider merging both ifs
  `if (www_basicauth_enabled && !server.authenticate(www_username, www_password))`